module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -935,7 +935,7 @@ public function showShardCheckboxes()
 
     /**
      * If there is a limit to how many search results a user can access, this
-     * method will return that limit.  If there is no limit, this will return
+     * method will return that limit. If there is no limit, this will return
      * -1.
```

Comment:
We could maybe move '-1' to the previous line. It would be more readable I think.